src/test/script_P2SH_tests.cpp

```diff
@@ -53,6 +54,7 @@ BOOST_FIXTURE_TEST_SUITE(script_P2SH_tests, BasicTestingSetup)
 BOOST_AUTO_TEST_CASE(sign)
 {
     LOCK(cs_main);
+    SelectPolicy("standard");
```

Comment:
Maybe after the rebase there's some redundancies with SelectPolicy()... @laanwj IIRC you recently changed the places where SelectParams() was called, can you confirm this is now redundant?